srv/modules/runners/validate.py

```diff
@@ -771,23 +771,18 @@ def _check_installed(self):
         install only happens once.
         """
         search = __utils__['deepsea_minions.show']()
-        results = self._silent_search(search, 'pkg.info_installed')
+        results = self._silent_search(search, 'pkg.list_pkgs')
 
-        for minion in results:
-            if isinstance(results[minion], dict) and self.package in results[minion]:
-                if 'version' in results[minion][self.package]:
-                    version = self._check_version(minion, 'pkg.info_installed',
-                                                  results[minion][self.package]['version'])
-                    if (version and LooseVersion(version) < LooseVersion(LUMINOUS_VERSION)):
-                        prefix = 'Ceph version is older than Luminous on'
-                        self.errors.setdefault('ceph_version', [prefix]).append(minion)
-                else:
-                    # Something is really wrong
-                    prefix = 'Version missing from'
+        for minion, pkg_list in results.items():
+            if self.package in pkg_list:
+                version = pkg_list[self.package]
+                if (version and LooseVersion(version) < LooseVersion(NAUTILUS_VERSION)):
+                    prefix = 'Ceph version is older than Nautilus on'
```

Comment:
Do we need to make a similar change in `_check_available` ?

Reply:
We may end up with only a _check_available() call which internally calls 'info_available' and not 
the malfunctioning 'info_installed'.
Instead of checking if ceph-common is installed, we'd just install it and then use the 'info_installed' as it's intended to check for the supported versions.. 

but that needs some thoughts.. Currently this PR is to test if bsc#1125015 can be fixed with getting rid of 'info_installed'